packages/icons/src/components/CryptoIconWithPercentage.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import { Canvas, Circle, Group, ImageSVG, Path, Skia, useSVG } from '@shopify/react-native-skia';
+
+import { useStyles } from '@trezor/styles';
+import { currencyColors } from '@trezor/theme';
+
+import { CryptoIconName, cryptoIcons } from '../icons';
+
+const CANVAS_SIZE = 48;
+const ICON_SIZE = 24;
+const RADIUS = 20;
+
+type CryptoIconProps = {
+    iconName: CryptoIconName;
+    percentage: number;
+    percentualOffset: number;
+};
+export const CryptoIconWithPercentage = ({
+    iconName,
+    percentage,
+    percentualOffset,
+}: CryptoIconProps) => {
+    const iconSvg = useSVG(cryptoIcons[iconName]);
+    const { utils } = useStyles();
```

Comment:
`useStyles` is for web usage, it works here because you don't use `applyStyle`, but use correct `useNativeStyles` please.